app/api/v1/api.py

```diff
@@ -722,35 +722,31 @@ async def api_get_score_info(
 @router.get("/get_replay")
 async def api_get_replay(
     score_id: int = Query(..., alias="id", ge=0, le=9_223_372_036_854_775_807),
-    include_headers: bool = False,
+    include_headers: bool = True,
```

Comment:
is there a particular reason this default changed? i'm not arguing for or against the default being True, but rather the fact it is a change in behaviour

Reply:
we don't, currently, need to utilise include_headers enough to explicitly declare it True every time, since in 99/100 cases when we use this endpoint, it is a frontend grabbing a replay

it simplifies the use of this endpoint, and as a result, users won't have to explicitly declare include_headers=True, since it will already be default for True